src/analyses/dependence_graph.h

```diff
@@ -162,7 +162,9 @@ class dep_graph_domaint:public ai_domain_baset
   node_indext node_id;
 
   typedef std::set<goto_programt::const_targett> depst;
-  depst control_deps, data_deps;
+  depst control_deps;
+  depst control_dep_candidates;
```

Comment:
It would be good to document here what the difference between a control dependency and a control dependency candidate is